src/content/guides/getting-started.md

```diff
@@ -22,11 +22,12 @@ contributors:
   - byzyk
   - wizardofhogwarts
   - myshov
+  - jamesgeorge007
 ---
 
 webpack is used to compile JavaScript modules. Once [installed](/guides/installation), you can interface with webpack either from its [CLI](/api/cli) or [API](/api/node). If you're still new to webpack, please read through the [core concepts](/concepts) and [this comparison](/comparison) to learn why you might use it over the other tools that are out in the community.
 
-W> Since webpack v5.0.0-beta.1 the minimum Node.js version to run webpack is 10.13.0 (LTS)
+W> Since [webpack v5.0.0-beta.1](https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.1) the minimum Node.js version to run webpack is 10.13.0 (LTS)
```

Comment:
Not sure if its worth it, wdyt @montogeek ?
I really dont want these webpack version mentions to turn into link fiesta

Reply:
Uhmm, it is always nice to have the link to check its changelog :)
But yeah, we have a lot of those messages :/

Reply:
Should we keep it?